execute/plugin_e2e_test.go

```diff
@@ -0,0 +1,216 @@
+package execute
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/mock"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/libocr/commontypes"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	libocrtypes "github.com/smartcontractkit/libocr/ragep2p/types"
+
+	"github.com/smartcontractkit/chainlink-ccip/internal/libs/testhelpers"
+	"github.com/smartcontractkit/chainlink-ccip/internal/mocks"
+	"github.com/smartcontractkit/chainlink-ccip/internal/mocks/inmem"
+	"github.com/smartcontractkit/chainlink-ccip/internal/reader"
+	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
+	"github.com/smartcontractkit/chainlink-ccip/plugintypes"
+)
+
+func TestPlugin(t *testing.T) {
+	ctx := context.Background()
+	lggr := logger.Test(t)
+
+	nodeSetups := setupSimpleTest(ctx, t, lggr, 1, 2)
+	//runner := testhelpers.NewOCR3Runner(nodes, nodeIDs, o)
+
+	nodesSetup := nodeSetups
+	nodes := make([]ocr3types.ReportingPlugin[[]byte], 0, len(nodesSetup))
+	for _, n := range nodesSetup {
+		nodes = append(nodes, n.node)
+	}
+
+	nodeIDs := make([]commontypes.OracleID, 0, len(nodesSetup))
+	for _, n := range nodesSetup {
+		nodeIDs = append(nodeIDs, n.node.reportingCfg.OracleID)
+	}
+
+	//o, err := tc.initialOutcome.Encode()
+	//require.NoError(t, err)
+	runner := testhelpers.NewOCR3Runner(nodes, nodeIDs, nil)
+
+	res, err := runner.RunRound(ctx)
+	fmt.Println(res, err)
```

Comment:
I believe you want to do some assertions instead of just printing `res, err`

Reply:
Yes, I'm adding these in a followup PR.